pkg/rewrites/rewrites.go

```diff
@@ -0,0 +1,83 @@
+// Package rewrites provides primitives for working with rewrite rules.
+package rewrites
+
+import (
+	"nanotube/pkg/conf"
+	"nanotube/pkg/rec"
+	"regexp"
+
+	"github.com/pkg/errors"
+)
+
+// Rewrites represent all the routing rewrites/routing table.
+type Rewrites []Rewrite
+
+// Rewrite is a routing rewrite.
+type Rewrite struct {
+	From string
+	To   string
+	Copy bool
+
+	CompiledFrom *regexp.Regexp
```

Comment:
Is this "CompiledFrom" or "CompiledForm", I wonder...

Reply:
See above To.

Reply:
So, it's like compiled form of From?